roadmap.md

```diff
@@ -1,84 +1,59 @@
-However [we are revisiting PipelineResources](#task-interfaces-and-pipelineresources),
-and likely this `PipelineResource` will become one or more `Tasks` in
-[the catalog](https://github.com/tektoncd/catalog) instead. Once that happens,
-we will need to make decisions about how to release and maintain the
-supporting code.
+# Tekton Pipelines 2021 Roadmap
+
+This is an incomplete list of work we hope to tackle in 2021!
+
+* [The Tekton mission and vision](https://github.com/tektoncd/community/blob/main/roadmap.md#mission-and-vision)
+* [Roadmap items tracked via issues](https://github.com/tektoncd/pipeline/labels/area%2Froadmap)
+
+## Roadmap
+
+* [v1 API](https://github.com/tektoncd/pipeline/issues/3548)
+  * Support alpha fields within v1 and beta types ([TEP-0033](https://github.com/tektoncd/community/blob/main/teps/0033-tekton-feature-gates.md))
+* When expressions:
+  * Option to skip Task only ([TEP-0007](https://github.com/tektoncd/community/blob/main/teps/0007-conditions-beta.md#skipping-1))
+  * [Support in Finally Tasks](https://github.com/tektoncd/pipeline/issues/3438) ([TEP-0045](https://github.com/tektoncd/community/blob/main/teps/0045-whenexpressions-in-finally-tasks.md))
+* [Env vars at runtime](https://github.com/tektoncd/pipeline/issues/1606)
+* Workspaces/PVCs/Affinity Assistant ([TEP-0046](https://github.com/tektoncd/community/pull/318)
+  * [Investigate custom scheduler for PVCs](https://github.com/tektoncd/pipeline/issues/3052)
+* [OCI bundles to beta](https://github.com/tektoncd/pipeline/issues/3661) (and then v1)
+* Expanded expression support via [CelRun Task](https://github.com/tektoncd/pipeline/issues/3149)
+* [Pipelines in Pipelines](https://github.com/tektoncd/pipeline/issues/2134)
+* Support for more complex failure scenarios
+  * Allow task failure ([TEP-0050](https://github.com/tektoncd/community/pull/342))
+  * Allow step failure ([TEP-0040](https://github.com/tektoncd/community/pull/302))
+  * Decisions regarding more complex graph construction on failure
+* [Instrument Tekton resources](https://github.com/tektoncd/pipeline/issues/2814)
+  * Minimize overhead of running a pipeline
+* Improve ability to compose Tasks with Tasks ([TEP-0044](https://github.com/tektoncd/community/pull/316)) 
+* [Workspaces “from” other Tasks (express resource dependencies on workspaces)](https://github.com/tektoncd/pipeline/issues/3109)
+* [Custom Tasks](https://github.com/tektoncd/community/blob/main/teps/0002-custom-tasks.md) completion:
+  * [Pipeline Results](https://github.com/tektoncd/pipeline/issues/3595)
+  * Example controller for folks who want to create custom tasks
+  * Experimental custom tasks promotion (e.g. [CELRun](https://github.com/tektoncd/experimental/tree/master/cel)):
+    * Plan around how to promote (what requirements, process)
+    * Continuous integration + release automation
+    * Documentation and examples at tekton.dev
+    * Integration with operator
```

Comment:
hmmm i wonder if this actually belongs in the tekton wide roadmap instead 🤔  probably doesn't matter too much as long as it's captured somewhere.